src/main/graphql-server/helpers/cacheInitializer.js

```diff
@@ -0,0 +1,28 @@
+import ApolloClient from 'apollo-client'
+import { SchemaLink } from 'apollo-link-schema'
+import { InMemoryCache } from 'apollo-cache-inmemory'
+import { makeExecutableSchema } from 'graphql-tools'
+
+import typeDefs from '../graphql/typeDefs'
+import resolvers from '../graphql/resolvers'
```

Comment:
Remove these imports

Reply:
Is this file used?

Reply:
It's not used currently, but thought it might be helpful once we get the rehydration from server->client working with the apollo cache. Might be able to throw in this logic into what was merged in from https://github.com/connexta/revelio/pull/46